### ISSUE TITLE
/images/drawio/base64 served over http

### ISSUE THREAD

**Issue by ppff** [BookStackApp/BookStack#1694]
Context:
 - bookstack 0.27.4 in docker
 - http instance + ssl terminating reverse proxy

Description: the call to `https://<domain>/images/drawio/base64` is redirected to `http://<domain>/images/drawio/base64`. Hence, it is blocked by the browser as mixed content.

1) I'm not sure if that's because of my proxy setup, I have however never seen this bug with anything else.
2) I'm not sure if that's related to it but my drawing won't load, it will only display the loading gif.

**Comment by ssddanbrown** [comment]
Hi @ppff,
Do you have the `APP_URL` environment variable set?

**Comment by ppff** [comment]
Yes. This looks very specific, and a bit odd. I'll let you know if I find anything!

**Comment by ssddanbrown** [comment]
Since there's been no follow-up I'm going to close this. If the issue remains and is something you still require to be fixed please open a new issue, referencing this one.
